src/dlgTriggerEditor.cpp

```diff
@@ -7089,6 +7128,71 @@ void dlgTriggerEditor::slot_toggle_active()
     }
 }
 
+void dlgTriggerEditor::slot_move_source_find()
+{
+    int x = mpSourceEditorEdbee->width() - mpSourceEditorFindArea->width();
+    int y = mpSourceEditorEdbee->height() - mpSourceEditorFindArea->height();
+    if (mpSourceEditorEdbee->verticalScrollBar()->isVisible())
```

Comment:
Put single-line if statements in braces - while it is a bit of a chore, it prevents [stupid mistakes](https://nakedsecurity.sophos.com/2014/02/24/anatomy-of-a-goto-fail-apples-ssl-bug-explained-plus-an-unofficial-patch/)

Reply:
Interesting read =o

Reply:
Fixed in commit bd22fd0